nym-wallet/src/pages/auth/pages/signin-mnemonic.tsx

```diff
@@ -18,7 +18,7 @@ export const SignInMnemonic = () => {
     setPasswordExists(hasPassword);
   };
 
-  const handlePageChange = (page: string) => {
+  const handlePageChange = (page: any) => {
```

Comment:
```suggestion
  const handlePageChange = (page: string | number) => {
```

Reply:
I did try this before but I get a type error. I think we'll need to add the react router types package to get around this